docs/IR.md

```diff
@@ -366,7 +366,7 @@ _Historical Notes_: The following extensions were considered early on, but were
 
 ### Attribute Types
 
-The type system used for attributes is a superset of that used for of inputs and outputs. In addition to tensors, attribute values may be scalar numerical values, strings, and graphs. Sequences are available for attributes in both ONNX and ONNX-ML. Maps are not available for attributes in either variant. 
```

Comment:
I took this to refer to `AttributeProto.tensors`, but I suppose `tensors` allows heterogeneous sequences, where sequence values are homogenous in value type, and will probably be added in #2548.

Maybe there should be something said about allowing multiple tensors and graphs in one attribute?

Reply:
"a superset of" seems incorrect too. How about replacing it by "related to but slightly different from"?

Reply:
I also took "sequences" to refer to the "repeated" version of all attributes. I also note that there is a PR in progress (https://github.com/onnx/onnx/pull/2581 ) to define the message format for Sequences and Maps. It does not yet seem to add this to AttributeProto, but I assume that is the intention or end-goal.

Reply:
I am also not clear what "either variant" refers to ... sounds like a leftover error. Why not simplify this to:
```
The type system used for attributes is related to but slightly different from that used for of inputs and outputs.
Attribute-values may be a dense tensor, or sparse tensor, or a scalar numerical value, or a string, or a graph, or
repeated values of one of the abovementioned types.
```

Reply:
Currently, some ops encode a map as a pair of attributes (repeated keys, repeated values).

Reply:
I am in favor of the wording @gramalingam mentioned above. Definitely adds clarity.